### ISSUE TITLE
debridnet.com

### ISSUE THREAD

**Issue by uBlock-user** [uBlockOrigin/uAssets#302]
### URL(s) where the issue occurs

http://debridnet.com/

### Describe the issue

website redirects the user to http://debridnet.com/ad-blocker

### Screenshot(s)

![](https://i.gyazo.com/11e290086edb4fa8d62dd74476e94686.png)

### Versions

- Browser/version: Chromium 58
- uBlock Origin version: 1.11.0

**Comment by okiehsch** [comment]
`||amazon-adsystem.com^$subdocument,redirect=noopframe,domain=debridnet.com`
works on my end with Chrome 56 and Firefox.

**Comment by okiehsch** [comment]
You lost me. Why can't the uMatrix user keep the `amazon-adsystem.com` domain blacklisted?
And the uMatrix user can just block first party iframes anyway.

![enannt](https://cloud.githubusercontent.com/assets/16838044/23524924/ed5a0f84-ff8c-11e6-9b25-911a3155c4e4.JPG)

**Comment by gorhill** [comment]
> That allows the uMatrix users to keep

No uBO filters should be added for the purpose of other extensions. If someone is using uMatrix (or whatever other extensions), and it interfere with uBO properly dealing with a site, then a solution specific to that setup must be figured by the user.

**Comment by uBlock-user** [comment]
> Why can't the uMatrix user keep the amazon-adsystem.com domain blacklisted?

Because anti adblock gets detected since uMatrix blocks the frames and doesn't redirect it like uBO does as it doesn't have such functionality.

> And the uMatrix user can just block first party iframes anyway.

No, I wouldn't do that for all websites as now-a-days- majority of them have video players which will not load when visiting any article or a page, a website specific block will work though. 

>  If someone is using uMatrix (or whatever other extensions), and it interfere with uBO properly dealing with a site, then a solution specific to that setup must be figured by the user.

Sure, I already did that for myself. I only suggested it as there are quite a number of folks like me who use uBO + uMX and adding that filter will benefit us too and not just uBO users, but it's your call.
